README.md

```diff
@@ -1,3 +1,10 @@
+## Lab 7 David
```

Comment:
-1 point
Doc instructions also say to write instructions on how to use your routes.